@css/main_style.css

```diff
@@ -44,52 +48,246 @@ title {
 body {
   color: blue;
   background-color: #fff;	/* #ffffff */
+  margin: 0;
+  padding: 0;
 }
 
 h1 {
-  color: white;
-  background-color: #00f;	/* #0000ff */
+  color: inherit;
+  background-color: inherit;
+}
+
+h2 {
+  color: inherit;
+  background-color: inherit;
+}
+
+h3 {
+  color: inherit;
+  background-color: inherit;
+}
+
+h4 {
+  color: inherit;
+  background-color: inherit;
 }
 
 p {
-  color: white;
-  background-color: #00f;	/* #0000ff */
+  color: inherit;
+  background-color: inherit;
+}
+
+ul {
+  color: inherit;
+  background-color: inherit;
+}
+
+li {
+  color: inherit;
+  background-color: inherit;
+}
+
+/* ******************* *
+ * CSS Media Selectors *
+ * ******************* */
+
+@media (min-width: 375px) and (max-width: 768px) {
+  header {
+    left: 0;
+    right: 0;
+  }
+}
+
+@media (min-width: 769px) {
+  header {
+    box-sizing: border-box;
+    left: 0;
+    width: 768px;
+  }
+
+  body {
+    width: 768px;
+  }
 }
 
+/* ******************** *
+ * Semantic HTML 5 tags *
+ * ******************** */
+
 header {
-  color: white;
-  background-color: blue;
+  color: inherit;
+  background-color: inherit;
+  position: fixed;
+  left: 0;
+  right: 0;
+  margin: 0;
+  padding: 10px;
 }
 
 nav {
-  color: white;
-  background-color: grey;
+  color: inherit;
+  background-color: inherit;
 }
 
+/* ********************* *
+ * Compund CSS Selectors *
+ * ********************* */
+
 nav > ul {
   padding: 0;
 }
 
 nav > ul > li {
+  color: inherit;
+  background-color: inherit;
   list-style: none;
   padding-left: 3em;
   padding-right: 3em;
 }
 
+div.main-body {
+  min-height: 100vh;
+}
+
+#section_1 > div > ul {
+  padding-inline-start: 0;
+  column-gap: 15px;
+  margin: 0;
+}
+
+#section_1 > div > ul > li {
+  row-gap: 15px;
+}
+
 /* ***************
  *  CSS Classes *
 ** **************/
 
+.w-375px {
+  width: 375px;
+}
+
+.h-750px {
+  height: 750px;
+}
+
 .flex-box {
   display: flex;
   align-content: center;
   justify-content: space-between;
 }
 
+.block-box {
+  display: block;
+}
+
+.inline {
+  display: inline;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.invisible {
+  display: none;
+}
+
+/* FLEX classes */
+
+.horizontal {
+  flex-direction: row;
+}
+
+.vertical {
+  flex-direction: column;
+}
+
+.flex-start {
+  justify-content: flex-start;
+}
+
+.flex-end {
+  justify-content: flex-end;
+}
+
+.flex-center {
+  justify-content: center;
+}
+
+.flex-space-between {
+  justify-content: space-between;
+}
+
+.flex-space-around {
+  justify-content: space-around;
+}
+
+.flex-space-evenly {
+  justify-content: space-evenly;
+}
+
+/* END of FLEX xlasses */
+
+.no-bullet {
+  list-style: none;
+}
+
 .logo {
   padding: 10px;
 }
 
 .nav-bar {
   /* justify-content: flex-end; */
```

Comment:
- [ ] [OPTIONAL] Good work on this project so far :clap: but please, for this line, I hope you have not forgotten how important it is not to leave commented codes in our work base as best practices have it. Kindly remove the code completely if it is no longer useful, including the empty declaration too :+1: